apps/etterlatte-statistikk/src/main/resources/db/migration/V19__add_sak_columns_stoenad.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE stoenad ADD COLUMN sak_utland TEXT;
+ALTER TABLE maaned_stoenad ADD COLUMN sak_utland TEXT;
```

Comment:
kan vurdere en patching av gammel data med 

`UPDATE stoenad SET sak_utland = 'NASJONAL' WHERE sak_utland IS NULL;` og 
`UPDATE maaned_stoenad SET sak_utland = 'NASJONAL' WHERE sak_utland IS NULL;`